code/game/area/Space Station 13 areas.dm

```diff
@@ -214,52 +214,4 @@ area/space/atmosalert()
 	luminosity = 1
 	dynamic_lighting = 0
 	requires_power = 0
-	var/sound/mysound = null
-
-/area/beach/New()
-	..()
-	var/sound/S = new/sound()
-	mysound = S
-	S.file = 'sound/ambience/shore.ogg'
-	S.repeat = 1
-	S.wait = 0
-	S.channel = GLOB.sound_channels.RequestChannel(/area/beach)
-	S.volume = 100
-	S.priority = 255
-	S.status = SOUND_UPDATE
-	process()
-
-/area/beach/Entered(atom/movable/Obj,atom/OldLoc)
-	. = ..()
-	if(ismob(Obj))
-		var/mob/M = Obj
-		if(M.client)
-			mysound.status = SOUND_UPDATE
-			sound_to(M, mysound)
-
-/area/beach/Exited(atom/movable/Obj)
-	. = ..()
-	if(ismob(Obj))
-		var/mob/M = Obj
-		if(M.client)
-			mysound.status = SOUND_PAUSED | SOUND_UPDATE
-			sound_to(M, mysound)
-
-/area/beach/proc/process()
-	set background = 1
-
-	var/sound/S = null
-	var/sound_delay = 0
-	if(prob(25))
-		S = sound(file=pick('sound/ambience/seag1.ogg','sound/ambience/seag2.ogg','sound/ambience/seag3.ogg'), volume=100)
-		sound_delay = rand(0, 50)
-
-	for(var/mob/living/carbon/human/H in src)
-		if(H.client)
-			mysound.status = SOUND_UPDATE
-			to_chat(H, mysound)
-			if(S)
-				spawn(sound_delay)
-					sound_to(H, S)
-
-	spawn(60) .()
+	forced_ambience = list('sound/ambience/seag1.ogg','sound/ambience/seag2.ogg','sound/ambience/seag3.ogg')
```

Comment:
I'll merely note that the (forced) ambiance system will pick one ambiance and play it on repeat forever, unlike the replaced one that selects a new sound every 6 seconds (not considering the 25% probability-thing)

Reply:
Having listened to these on endless repeat with no pause between, just no.

Reply:
Back to old system.